Laravel Blade.sublime-syntax

```diff
@@ -4,7 +4,7 @@ name: Laravel Blade
 file_extensions:
   - blade
   - blade.php
-scope: text.blade
+scope: source.blade
```

Comment:
The issue with this change is that you can't expand `div.foo` to `<div class="foo></div>`, hence why I changed it back. Unless we can use a `meta_content_scope`? I don't know.